net-p2p/bitcoind/bitcoind-0.14.2.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2010-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+BITCOINCORE_COMMITHASH="fc61c8322bd7288f7546d18ad04c36c345be13cd"
+BITCOINCORE_LJR_DATE="20170618"
+BITCOINCORE_IUSE="bip148 examples knots no-bip148 test upnp +wallet zeromq"
+BITCOINCORE_POLICY_PATCHES="+rbf spamfilter"
+BITCOINCORE_NEED_LEVELDB=1
+BITCOINCORE_NEED_LIBSECP256K1=1
+inherit bash-completion-r1 bitcoincore user systemd
+
+DESCRIPTION="Original Bitcoin crypto-currency wallet for automated services"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux ~arm ~arm64 ~mips ~ppc ~x86 ~x86-linux"
+
+REQUIRED_USE="^^ ( bip148 no-bip148 )"
```

Comment:
"You can't" is not really an option here. It's not acceptable to just make decisions for people that put their finances at risk (both options have such risks). If PMs are going to make a decision on REQUIRED_USE, then I guess it can be removed and replaced with a `use bip148 || use no-bip148 || die`...